components/ForgotPassword.tsx

```diff
@@ -0,0 +1,119 @@
+"use client";
+
+import Image from "next/image";
+import { useState } from "react";
+
+// Logo Import
+import logo from "../public/Cacart_logo.png";
+
+// Image Import
+import forgotPasswordImage from "../public/forgotPassword.png";
+import dottedShapeImage from "../public/Dotted Shape.png";
+
+export default function ForgotPassword() {
+  const [userData, setUserData] = useState({
```

Comment:
We are dealing with 1 piece of information (email), but an object is used here. What would be the benefits of this?

Reply:
for Scalability...?  😅 I think it's better to change it to a string state, as you suggested.

Reply:
DONE